components/online_image.rst

```diff
@@ -0,0 +1,134 @@
+.. _online_image:
+
+Online Image Component
+======================
+
+.. seo::
+    :description: Instructions for displaying images downloaded at runtime in ESPHome.
+    :image: image-sync-outline.svg
+
+With this component you can define images that will be downloaded, decoded and drawn at runtime.
+
+.. note::
+
+    Currently only images in PNG format are supported.
+
+.. warning::
+
+    This component requires a fair amount of RAM; both for downloading the image, and for storing the decoded image. It might work on devices without PSRAM, but there is no guarantee.
+
+This component has a dependency to :doc:`/components/http_request`; the configuration options you set to the ``http_request`` component will also apply here.
+
+.. code-block:: yaml
+
+    http_request:
+
```

Comment:
It's on the user to visit the above link to see how to configure the `http_request` component
```suggestion
```